.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,105 @@
+name: Integration Tests
+
+on:
+  push:
+    branches:
+      - main
+      - release/**
+  pull_request:
+
+jobs:
+
+  # We borrow the tests from the Google I/O Android App under the Apache license: https://github.com/google/iosched/.
+  # The following steps checkout iosched and apply a github patch to the project. The patch adds
+  # Sentry to the app with auto performance monitoring enabled. We then run the UI tests to make sure
+  # adding our SDK doesn't cause any major issues.
+  build-iosched:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'google/iosched'
+          ref: '6f96c1528c00d3d89bedb33e5a0a10d67629e8bd'
```

Comment:
Why this specific ref?

Reply:
If google/iosched gets a flaky test on the master branch, I don't want this to affect our CI. We have to manually update the ref from time to time.

Reply:
Added a comment.